e2e/test/Helpers/StorageContainer.cs

```diff
@@ -129,21 +125,23 @@ protected virtual void Dispose(bool disposing)
                 return;
             }
 
-            if (disposing && BlobContainerClient != null)
+            if (disposing && CloudBlobContainer != null)
             {
-                BlobContainerClient.Delete();
-                BlobContainerClient = null;
+                CloudBlobContainer.Delete();
+                CloudBlobContainer = null;
             }
 
             _disposed = true;
         }
 
         private async Task InitializeAsync()
         {
-            BlobContainerClient = new BlobContainerClient(Configuration.Storage.ConnectionString, ContainerName);
-            await BlobContainerClient.CreateIfNotExistsAsync().ConfigureAwait(false);
+            _ = CloudStorageAccount.TryParse(Configuration.Storage.ConnectionString, out CloudStorageAccount storageAccount);
```

Comment:
not familiar with the '_' syntax. What does this do?

Reply:
It means discard the output from this method. I don't care about it. It signals to any code reader that the return parameter was not ignored by accident.

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/discards

Reply:
I checked and there is a Parse method instead, so I'll change to use that.